apps/aehttp/priv/oas3.yaml

```
@@ -3285,6 +3285,14 @@ components:
       required:
         - key
         - id
+    NamePointerComplete:
```

as alternative naming I can suggest
```suggestion
    NamePointerOut:
```
and `NamePointerIn` above